recruiter/models.py

```diff
@@ -1,3 +1,23 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Company(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(max_length=500)
+    website_url = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Recruiter(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    first_name = models.CharField(max_length=255)
```

Comment:
Why do you need to separate the first and last names? Not all names have just 2 components, see similar comment I made on the Student model PR.